.travis/setup.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 if [[ $TRAVIS_OS_NAME == "osx" ]]; then
-    brew install openmpi  # use openmpi to install gcc
+    brew install gcc --without-multilib
```

Comment:
`--without-multilib` may be not needed for tests

Reply:
@henry0312 I will merge this first. you can test the one without ```--without-multilib``` later.